node/overseer/src/lib.rs

```diff
@@ -654,6 +664,9 @@ impl ChannelsOut {
 			AllMessages::GossipSupport(msg) => {
 				self.gossip_support.send(make_packet(signals_received, msg)).await
 			},
+			AllMessages::DisputeDistribution(msg) => {
+				self.dispute_distribution.send(make_packet(signals_received, msg)).await
```

Comment:
Worth noting that other dispute subsystems aren't yet wired up. Does distribution handle that gracefully? We might want to wire them all up at the same time.

Reply:
Very likely not, will remove the wiring for now. Thanks for the catch!